pkg/operator/staticpod/controller/installer/installer_controller.go

```diff
@@ -422,15 +422,77 @@ func setNodeStatusFn(status *operatorv1.NodeStatus) v1helpers.UpdateStaticPodSta
 	}
 }
 
+type revisionDescriptionPrinter struct {
+	nodes          []operatorv1.NodeStatus
+	latestRevision int32
+	progressing    bool
+}
+
+func newRevisionDescriptionPrinter(nodes []operatorv1.NodeStatus, latestRevision int32, progressing bool) *revisionDescriptionPrinter {
+	return &revisionDescriptionPrinter{nodes: nodes, latestRevision: latestRevision, progressing: progressing}
+}
+
+func pluralizedNode(i int) string {
+	if i < 2 {
+		return "node"
+	}
+	return "nodes"
+}
+
+func (r revisionDescriptionPrinter) String() string {
+	currentRevisions := sets.NewInt32()
+	latestCount := 0
+	availableCount := 0
+	for i := range r.nodes {
+		c := r.nodes[i].CurrentRevision
+		currentRevisions.Insert(c)
+		if c == r.latestRevision {
+			latestCount++
+		}
+		if c != 0 {
+			availableCount++
+		}
+	}
+
+	notAvailable := fmt.Sprintf("%d %s are not available and ", availableCount, pluralizedNode(availableCount))
```

Comment:
cannot follow. This must be `len(nodes)-availableCount`, no?

And better turn the wording into: `%d %s have never been available and`